lib/ses-to-cloudwatch/plugin.php

```diff
@@ -72,11 +96,10 @@ function send_event_to_stream( array $event, string $group, string $stream ) {
 	if ( $next_token ) {
 		$params['sequenceToken'] = $next_token;
 	}
-	$promise = cloudwatch_logs_client()->putLogEventsAsync( $params )
```

Comment:
Why no more asyc?

Reply:
The way "async" actually works is by stacking up requests, and then executing all with a `wait()` on the promise. when sending single requests, this is actually no faster. `wait()` blocks the PHP process for as long the request takes, it's not like an async JS request.